docs/source/api_reference/pydax.dataset.rst

```diff
@@ -0,0 +1,17 @@
+pydax.dataset
```

Comment:
Did you create these files completely manually? Is there something you can add to the CONTRIBUTING file regarding generating these files?

Reply:
Yeah I created everything in `docs/source/api_reference/*.rst` and they generate all the stub files to `docs/source/api_reference/autosummary`